src/features/MoviesBrowser/SearchResult/index.js

```diff
@@ -38,24 +48,34 @@ const SearchResult = ({ onPageChange, currentPage }) => {
         title={
           isError || !data || data.results.length === 0
             ? `Sorry, there are no results for "${queryParam || ""}"`
-            : `Search result for "${queryParam || ""}" (${data?.results.length || 0})`
+            : `Search result for "${queryParam || ""}" (${
+                data?.results.length || 0
+              })`
         }
       />
       {isLoading ? (
         <Loading />
-      ) : isError || !data || data.results.length === 0 ? (
+      ) : isError ||
+        !dataLoaded ||
+        !data ||
+        !data.results ||
+        data.results.length === 0 ? (
```

Comment:
Isn't it a bit too complicated? 😅

Reply:
I made an issue - agree, that it should have a little less restrictions :D